gradle/dependencies.gradle

```diff
@@ -7,7 +7,7 @@ ext.versions = [
     minSdk: 14,
     autoValue: '1.6.2',
     schemaCrawler: '14.16.04.01-java7',
-    sqldelight: '1.0.0-alpha4',
+    sqldelight: '1.0.0-SNAPSHOT',
```

Comment:
happy to alpha this but maybe we should wait for 1.3 final?

Reply:
yeah let's wait